auth/service/service/service.go

```diff
@@ -39,6 +45,18 @@ func New() *Service {
 	}
 }
 
+func (s *Service) Run() error {
+	errs := make(chan error, 0)
```

Comment:
Does this leak channels, or is it called only once?

Reply:
Called once on startup